apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -4300,6 +4303,9 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
         if (this.depositAmount) {
             let deposit = cloneDeep(this.depositAmount);
             this.adjustPaymentBalanceDueData = this.adjustPaymentBalanceDueData - deposit;
+        }
+         if (this.adjustPaymentData && this.adjustPaymentData.totalAdjustedAmount) {
+            this.adjustPaymentBalanceDueData = this.adjustPaymentBalanceDueData + this.invFormData.voucherDetails.tcsTotal + this.invFormData.voucherDetails.tdsTotal;
```

Comment:
`this.adjustPaymentData.totalAdjustedAmount` is not subtracted in the expression
Consider creating a method for balance due data calculation as this code is repeating